packages/ckeditor5-utils/src/config.ts

```diff
@@ -250,6 +250,11 @@ export default class Config<Cfg> {
 			source = source[ part ];
 		}
 
+		// Return actual value if it's a function.
+		if ( source && typeof source[ name ] === 'function' ) {
+			return source[ name ];
+		}
```

Comment:
Maybe we could handle functions the same way as DOM node references (inside `cloneConfig()`)?